src/lib/pickles/step_verifier.ml

```diff
@@ -70,6 +70,7 @@ struct
       (Inner_curve.to_field_elements g2)
     |> Boolean.all
 
+  (** Absorbs `t` (of type `ty` into the sponge `sponge`. *)
```

Comment:
Don't use backticks in OCaml doc-comments, use square brackets.

Reply:
I thought square brackets were links to types that existed, whereas here they are arguments to the function